### PR TITLE
Return a new map from the statement service instead of the original

### PR DESCRIPTION
**Pull request by sroberts2**
Prevented a bug where this class from exposing the internal representation of the enclosed properties Map.

Resolves a bug on: SFA-829

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/api/accounts/service/impl/StatementsServiceProperties.java [hunk 1]
```diff
@@ -13,7 +14,7 @@
     private Map<String, String> statements;
 
     public Map<String, String> getStatements() {
-        return statements;
+        return new HashMap<>(statements);
```
Comment (ljgeorge) [inline src/main/java/uk/gov/companieshouse/api/accounts/service/impl/StatementsServiceProperties.java]: Hi, is this different to initialising  **statements** like this:
**private Map<String, String> statements = new HashMap<>();**
and then just returning **statements** from the **getStatements()** method ?

Comment (sroberts2) [reply src/main/java/uk/gov/companieshouse/api/accounts/service/impl/StatementsServiceProperties.java]: It creates a new clone of the Map every time `getStatements` is called preventing the original Map from being altered. I could rename the method `getCopyOfStatements` to make it clearer what the intent is?

Comment (ljgeorge) [reply src/main/java/uk/gov/companieshouse/api/accounts/service/impl/StatementsServiceProperties.java]: I think your suggestion for the method name is good, would getCloneOfStatements ? be more accurate?

Comment (sroberts2) [reply src/main/java/uk/gov/companieshouse/api/accounts/service/impl/StatementsServiceProperties.java]: Agreed I will change it now.

**Pull request merged**